### DISCUSSION TITLE
Feature Request: Datepicker month only

### DISCUSSION THREAD

**Discussion by rustyy** [payloadcms/payload#920] [Feature Requests & Ideas]
Hi there,
I played around a little bit with the date field and had the requirement to select the month only.

As of now only `dayAndTime`, `timeOnly` and `dayOnly` are supported. As react-datepicker supports [month selection](https://reactdatepicker.com/#example-month-picker) by configuration, it would be great having this supported by payload.

I created a feature branch here: https://github.com/rustyy/payload/tree/feat/datepicker-month-only 

Let me know about your thoughts.

<img width="1785" alt="Bildschirmfoto 2022-08-12 um 12 45 13" src="https://user-images.githubusercontent.com/1225568/184339331-46cd66f1-8899-4f8e-8455-4d5a6dc0563f.png">

**Comment by jmikrut** [comment]
This is great!

THANK YOU for the work involved here.

Will you PR? We'd be happy to merge!

**Reply by rustyy** [reply]
Hey @jmikrut and thank you too!
And for sure, PR is created #926 :)
